docker-compose.yml

```diff
@@ -16,9 +16,7 @@ services:
 
   web:
     build: .
-    command: bash -c "poetry install &&
-                      poetry run python manage.py compilemessages --ignore \".venv\" &&
-                      poetry run python manage.py runserver_plus 0.0.0.0:80"
+    command: bash -c "poetry run python manage.py compilemessages --ignore \".venv\" && poetry run python manage.py runserver_plus 0.0.0.0:80"
```

Comment:
If we don't run `poetry install` here and some packages are updated, we'd need to build the image again, right? 
It's a small thing but I feel it's slightly more convenient to just update the packages when the container starts?